scripts/preparepackages.js

```diff
@@ -9,9 +9,9 @@
 
 'use strict';
 
-const { Listr } = require( 'listr2' );
-const releaseTools = require( '@ckeditor/ckeditor5-dev-release-tools' );
-const { tools } = require( '@ckeditor/ckeditor5-dev-utils' );
+import { Listr } from 'listr2';
+import releaseTools from '@ckeditor/ckeditor5-dev-release-tools';
+import { tools } from '@ckeditor/ckeditor5-dev-utils';
```

Comment:
We are using `require` in this package in line 55. Please make sure that this works well with modules. 
![image](https://github.com/ckeditor/ckeditor5-react/assets/114912428/7db14b3c-49cf-48db-82b1-29974bb01151)